src/common/obj_bencher.cc

```diff
@@ -376,7 +376,17 @@ int ObjBencher::fetch_bench_metadata(const std::string& metadata_file,
   } else {
     *op_size = *object_size;
   }
-  unsigned ops_per_object = *object_size / *op_size;
+
```

Comment:
please remove these empty lines. see https://google.github.io/styleguide/cppguide.html#Vertical_Whitespace

Reply:
Thanks, will do.

Reply:
Okay, removed it is now

```
  ...
  if (!p.end()) {
    decode(*op_size, p);
  } else {
    *op_size = *object_size;
  }
  unsigned ops_per_object = 1;
  // make sure *op_size value is reasonable
  if (*op_size > 0 && *object_size > *op_size) {
    ops_per_object = *object_size / *op_size;
  }
  *num_objects = (*num_ops + ops_per_object - 1) / ops_per_object;
```